integration/mediation-tests/tests-patches/src/test/java/org/wso2/carbon/esb/mediators/rule/ESBJAVA2506RuleFetchFromRegistryFailsForTheFirstTime.java

```diff
@@ -23,36 +23,21 @@
 import org.testng.annotations.Test;
```

Comment:
License needs to be updated with the year created.

Reply:
It's already there :)